packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -658,6 +660,29 @@ export const Table = <
     [getPreparedRow],
   );
 
```

Comment:
Naming is weird... maybe just `dispatchStickyEvent` or something like that.. It does not really set any state here.

Reply:
Renamed